templates/repo/settings/webhook/history.tmpl

```diff
@@ -21,7 +21,7 @@
 						<a class="ui primary sha label toggle button show-panel" data-panel="#info-{{.ID}}">{{.UUID}}</a>
 						<div class="ui right">
 							<span class="text grey time">
-								{{.DeliveredString}}
+								{{TimeSince .Delivered.AsTime $.lcoale}}
```

Comment:
Typo `lcoale`?

Reply:
```suggestion
								{{TimeSince .Delivered.AsTime $.locale}}
```

yes, it's typo , and actually this argument is never used by backend, so no error occurs.